src/ZoneServer/Scripting/AI/AiScript.Routines.cs

```diff
@@ -291,7 +291,7 @@ protected IEnumerable Follow(ICombatEntity followTarget, float minDistance = 50,
 
 				if (catchUp)
 				{
-					var closePos = this.Entity.Position.GetRelative(followTarget.Position, 50);
+					var closePos = followTarget.Position;
```

Comment:
Will this not make the following actor always run to the target's exact position, and potentially glue companions and other followers to their masters?

Reply:
This doesn't happen because when they get close enough `isTargetInRange` will return true then the follower will stop catching up.

Also getting the relative position was not exactly correct because that position could be out of bounds and not exactly the position the follower should be in relation to the master.

Reply:
What exactly is concerning you or you think we could improve here?

Reply:
Don't worry about it, I'm overthinking unlikely potential issues. Like, a follower _could_ move fast enough to get close to the target before stopping for example. But in practice that will probably never happen.

Reply:
Well what we could try to do here is a GetRelative between the target's position and the _lastPathfindingGoal, since we know the pathfinding calculated position would be valid and on the right angle towards the target. It would remain the possibility of running out of bounds if not using pathfinding, though.

Reply:
I don't know if that's actually a concern, because we naturally want pathfinding to be on, and the option to disable it will most likely get removed eventually. Additionally, if we wanted to to keep it, we would then of course have to make sure monsters don't run through walls.